src/redux/book/bookSlice.js

```diff
@@ -0,0 +1,25 @@
+const ADDBOOK = 'book-store/redux/book/bookSlice/ADDBOOK';
+const REMOVEBOOK = 'book-store/redux/book/bookSlice/REMOVEBOOK';
+
```

Comment:
- [ ] In this activity we want _booksSlice_ to make use of `createSlice` from `@reduxjs/toolkit`, and should follow the below construct:
  * Imports `createSlice` from `@reduxjs/toolkit`
  * Creates an `initialState` with a state value `books` that contains an empty array
  * Creates 2 reducers, `addBook` and `removeBook`
  * Exports by default `booksSlice.reducer`
  * Exports by `addBook` and `removeBook` as actions

It should resemble the following code example:
```javascript
// ./src/redux/books/booksSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})

export const { addBook, removeBook } = booksSlice.actions

export default booksSlice.reducer
```